collector/src/toolchain.rs

```diff
@@ -123,6 +123,7 @@ impl SysrootDownload {
         let components = ToolchainComponents::from_binaries_and_libdir(
             sysroot_bin("rustc")?,
             Some(sysroot_bin("rustdoc")?),
+            Some(sysroot_bin("clippy")?),
```

Comment:
As is apparent on CI, `clippy` might not always be applicable. So we should only pass it here if it actually exists. Also, it should also probably be `cargo-clippy` here.

Reply:
Probably done, I'll check CI to see if the fix I did is correct :cat: